demo/sass/style.scss

```diff
@@ -1,30 +1,179 @@
-@import "./node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties";
+@import './node_modules/@alaskaairux/design-tokens/dist/tokens/SCSSVariables';
+@import './node_modules/@alaskaairux/design-tokens/dist/tokens/SassCustomProperties';
 
 // Import baseline library dependencies
 // Mixins
-@import "./node_modules/@alaskaairux/webcorestylesheets/dist/breakpoints";
+@import './node_modules/@alaskaairux/webcorestylesheets/dist/breakpoints';
 
 // Creates baseline output in CSS
-@import "./node_modules/@alaskaairux/webcorestylesheets/dist/fonts";
-@import "./node_modules/@alaskaairux/webcorestylesheets/dist/normalize";
-@import "./node_modules/@alaskaairux/webcorestylesheets/dist/essentials";
+@import './node_modules/@alaskaairux/webcorestylesheets/dist/fonts';
+@import './node_modules/@alaskaairux/webcorestylesheets/dist/normalize';
+@import './node_modules/@alaskaairux/webcorestylesheets/dist/essentials';
+@import './node_modules/@alaskaairux/webcorestylesheets/dist/headings';
 
 // Import Utility classes (will produce CSS in output)
 $layout-properties: margin;
 $layout-extensions: Right;
 $layout-modifiers: (
-  "md": $auro-size-md
+  'md': $auro-size-md
 );
-@import "./node_modules/@alaskaairux/webcorestylesheets/dist/utilityMixins/layoutPropertiesGenerator";
+@import './node_modules/@alaskaairux/webcorestylesheets/dist/utilityMixins/layoutPropertiesGenerator';
 
 // Custom styles per demo pages
-code {
-  color: var(--auro-color-base-crimson-500);
+body {
+  padding: 3rem;
 }
 
-.vertical-section-container {
+pre[class*='language-'] {
+  margin-top: 0;
+}
+
+main {
   margin: 0 auto;
-  max-width: 	82.5rem;
+  max-width: 60rem;
+}
+
+.exampleWrapper {
+  $stripe: #f3f3f3;
+  $background: var(--auro-color-base-white);
+
+  background: repeating-linear-gradient( 45deg, $stripe, $stripe 10px, $background 10px, $background 20px);
+  overflow: auto;
+}
+
+.exampleWrapper--ondark {
+  $stripe: var(--auro-color-background-darkest);
+  $background: var(--auro-color-background-darker);
+
+  background: repeating-linear-gradient( 45deg, $stripe, $stripe 10px, $background 10px, $background 20px);
+}
+
+// -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+
+:not(pre) > code[class*="language-"], pre[class*="language-"] {
+  background: #f8f8f8;
+}
+
+auro-accordion {
+  pre[class*="language-"] {
+    background: none;
+    margin-bottom: 0;
+    padding: 0;
+  }
+}
+
+code[class*="language-"], pre[class*="language-"] {
+  font-family: monospace;
+}
+
+// -=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=
+
+code:not(.html):not(.css):not(.js) {
+  color: var(--auro-color-brand-flamingo-500);
+}
+
+blockquote {
+  margin-left: 0;
+  border-left: 1px solid var(--auro-color-border-active-on-light);
+  padding: var(--auro-size-xs);
+  padding-left: var(--auro-size-xl);
+  color: var(--auro-color-text-secondary-on-light);
+  font-weight: 200;
+  margin-top: var(--auro-size-lg);
```

Comment:
can we tighten that top margin to 

var(--auro-size-md)

Reply:
This is the CSS to render the demo page, but I get what you are referring to. 

The margin around the type is `1rem`, so the update reduces the margin to `var(--auro-size-sm)`